Config/Moya.xcconfig

```diff
@@ -0,0 +1,11 @@
+
+OTHER_LDFLAGS[sdk=iphoneos13.0] = -weak_framework Combine
+OTHER_LDFLAGS[sdk=iphonesimulator13.0] = -weak_framework Combine
+
+OTHER_LDFLAGS[sdk=watchos6.0] = -weak_framework Combine
+OTHER_LDFLAGS[sdk=watchsimulator6.0] = -weak_framework Combine
+
+OTHER_LDFLAGS[sdk=appletvos13.0] = -weak_framework Combine
+OTHER_LDFLAGS[sdk=appletvsimulator13.0] = -weak_framework Combine
+
+OTHER_LDFLAGS[sdk=macosx10.15] = -weak_framework Combine
```

Comment:
Does this means we'll have to update this for every sdk update?

Reply:
I'm not entirely sure, but I think it means iOS 13+, so we won't have to update it

Reply:
what if we drop the config per sdk and just use `OTHER_LDFLAGS = -weak_framework Combine`? is there any drawback in linking Combine everywhere? will Xcode 10 be able to just ignore this statement?

Reply:
nvm just tried that and and it didn't ignore it but stopped the build... hmm...

Reply:
I tried, and for me, Xcode 10 failed to build it without the sdk specification

Reply:
unfortunately I tried adding a random weak framework to `sdk=iphonesimulator12.4` and it didn't build (as it should) on 12.4 but it built correctly on 13.0 so @amaurydavid raises a valid point - we would need to update that on each sdk release... is there any other way we could do the linking on 13+?

Reply:
hey @fredpi, was thinking, we can actually just make sure everyone uses Xcode 11 in the development process anyways - the only question is if it will work for people that use Moya in Xcode 10? did you try that maybe?

Reply:
In the SO answer I referenced, there's also another suggestion:

```
OTHER_LDFLAGS_XCODE_SPECIFIC_1100 = -weak_framework Combine

OTHER_LDFLAGS = $(inherited) ${OTHER_LDFLAGS_XCODE_SPECIFIC_$(XCODE_VERSION_ACTUAL)}
```

But it seems like that needs updating, too.

I also think, dropping Xcode 10 support altogether isn't feasible at this point of time...

Reply:
Yeah that doesn't seem as a better alternative. And yeah we don't want to remove support for Xcode 10 for our users for sure.

I'm just wondering if we remove the `sdk` constraint will it:
- break development in Xcode 10 (meaning Moya developers will have to use Xcode 11)
- break both development and client-side for Xcode 10